tests/5.0/target/test_target_default.c

```diff
@@ -0,0 +1,42 @@
+//===------ test_target_default.c --------------------------------------===//
```

Comment:
Since the default clause applies to the parallel, not target. The name should be test_target_parallel_default.c

Reply:
Changed the name of the file as suggested...